### PR TITLE
git-lfs 0.5.2

### PR DESCRIPTION
**Pull request by gurugray**
updated version section and shasum fo source files
removed bottle section for future correct shasums

### CODE REVIEW

**Comment by tgirardi** [comment]
+1

File: Library/Formula/git-lfs.rb [hunk 1]
```diff
@@ -1,15 +1,8 @@
...
   homepage "https://github.com/github/git-lfs"
-  url "https://github.com/github/git-lfs/archive/v0.5.1.tar.gz"
-  sha256 "8cefb145e47b2e582fc284d1fde5486f30e9c754f9d85d12db61c7397c797931"
-
-  bottle do
-    cellar :any
-    sha256 "796c25fb3b8efede11672dbf8dc87fb2c1ca7727dc96bb5e99e570eabe221ee5" => :yosemite
-    sha256 "f8325f948d1cfd67db4ad64674919d08a0763bfabda9a2743f022ad7d14d69ea" => :mavericks
-    sha256 "08d39395bc9107b37aeac9570f9bf225b624f19ad2b29bb416db44da06e7c975" => :mountain_lion
-  end
```
Comment (mikemcquaid) [inline Library/Formula/git-lfs.rb]: Please re-add this block; it's autoupdated when we pull. Thanks!

**Comment by DomT4** [comment]
```
==> ./script/bootstrap
script/build.go:13:2: cannot find package "github.com/github/git-lfs/lfs" in any of:
	/usr/local/Cellar/go/1.4.2/libexec/src/github.com/github/git-lfs/lfs (from $GOROOT)
	($GOPATH not set)
```

File: Library/Formula/git-lfs.rb (hunk 1 above)
Comment (gurugray) [inline Library/Formula/git-lfs.rb]: ok returned it

**Comment by mikemcquaid** [comment]
@gurugray This is still going to fail; did it work for you locally? Let's just use #41316 instead. Thanks!

**Comment by gurugray** [comment]
thanks!

**Comment by gurugray** [comment]
And i'm not go-developer, so I think it was my local misconfiguration

**Comment by mikemcquaid** [comment]
Yeh, it's confusing. Don't worry about it!

**Pull request closed without merging**